inst/shiny/modules/column_mapping.R

```diff
@@ -277,6 +277,11 @@ column_mapping_server <- function(id, data, manual_units, on_submit) {
     observeEvent(input$submit_columns, {
       Sys.sleep(1) # Make this artificially slow to show the loading spinner
 
+      # Enable other tabs
+      shinyjs::enable(selector = "#page li a[data-value=nca]")
+      shinyjs::enable(selector = "#page li a[data-value=visualisation]")
+      shinyjs::enable(selector = "#page li a[data-value=tlgs]")
```

Comment:
suggestion: In repetitive calls like this you can utilize some functional programming with `purrr`:
```suggestion
  purrr::walk(c("nca", "visualisation", "tlgs"), \(tab) {
    shinyjs::disable(selector = paste0("#page li a[data-value=", tab, "]"))
  })
```
but in this case, considering not much code repetition, it is absolutely fine.